index.html

```diff
@@ -43,5 +44,174 @@ <h1>I’m Alexandra<br>Glad to see you!</h1>
     <div class="sicons"></div>  
   </section>
 
+  <!-- Second Page -->
+  <section class="profile">
+    <ul class="exp">
+      <li class="profilexp">
+        <img class="banner" src="images/Snapshoot Portfolio.svg" alt="tannerbio">
+        <div class="mainfo">
+          <h2 class="tonic" id="title">Tonic</h2>
+          <div class="sub">
+            <h3 class="canopy">CANOPY</h3>
+            <ul class="experience">
+              <li class="bed"><a href="#">Back End Dev</a></li>
+              <li class="y2015"><a href="#">2015</a></li>
+            </ul>
+          </div>
+          <p class="about">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <div class="stack">
+            <p class="html">html</p>
+            <p class="css">css</p>
+            <p class="js">javaScript</p>
+          </div>
+          <div class="foot">
+            <button type="button" class="seep">See Project</button>
+          </div>
+        </div>
+      </li>
+      
+<!-- Third Page -->
+      <li class="profilexp">
+        <img class="banner" src="images/Snapshoot Availability.png" alt="Availability">
+        <div class="mainfo">
+          <h2 class="tonic" id="title">Multi-Post Stories</h2>
+          <div class="sub">
+            <h3 class="canopy">CANOPY</h3>
+            <ul class="experience">
+              <li class="bed"><a href="#">Back End Dev</a></li>
+              <li class="y2015"><a href="#">2015</a></li>
+            </ul>  
+          </div>
+          <p class="about">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <div class="stack">
+            <p class="html">html</p>
+            <p class="css">css</p>
+            <p class="js">javaScript</p>
+          </div>
+          <div class="foot">
+            <button type="button" class="seep">See Project</button>  
```

Comment:
- [ ] Kindly make changes as suggested by the previous reviewer. when clicked, it shows the hover color instead of the pressed color as shown on the template below. Kindly adjust this and you're good to go 👏